package.json

```diff
@@ -13,7 +13,7 @@
     "husky": "^1.0.0-rc.9",
     "lint-staged": "^7.2.0",
     "prettier": "^1.13.7",
-    "react-scripts": "1.0.10",
+    "react-scripts": "^1.1.5",
```

Comment:
You must use the exact version to ensure that everyone has the same working environment. Remove `^`